landing/index.html

```diff
@@ -1,1043 +1,1146 @@
-                        <ul>
-                           <li><a href="#header">Home</a></li>
-                           <li><a href="#features">Features</a></li>
-                           <li><a href="#video">Video</a></li>
-                           <li><a href="#screens">Screens</a></li>
-                           <li><a href="#testimonials">Testimonials</a></li>
-                           <li><a href="#pricing">Pricing</a></li>
-                           <li><a href="#team">Team</a></li>
-                           <li><a href="#blog">Blog</a></li>
-                           <li><a href="#freetrial">Free Trial</a></li>
-                           
-                        </ul>
-                     </div>
-                  </div>
-               </div>
+        <div class="row header-top" data-spy="affix" data-offset-top="70">
+          <div class="container">
+            <div class="row">
+              <div class="col-lg-3 col-md-3">
+                <div class="logo">
+                  <a href="#"><img src="images/logo.png" alt="logo"/></a>
+                </div>
+                <div id="menu-icon">
+                  <span></span><span></span><span></span>
+                </div>
+                <div class="clear"></div>
+              </div>
+              <div class="col-lg-9 col-md-9" id="menu">
+                <div class="menu">
+                  <ul>
+                    <li><a href="#header">Home</a></li>
+                    <li><a href="#features">Features</a></li>
+                    <li><a href="#video">Video</a></li>
+                    <li><a href="#screens">Screens</a></li>
+                    <li><a href="#testimonials">Testimonials</a></li>
+                    <li><a href="#pricing">Pricing</a></li>
+                    <li><a href="#team">Team</a></li>
+                    <li><a href="#newsletter-section">Newsletter</a></li>
+                  </ul>
+                </div>
+              </div>
             </div>
-            
-         </div>
-         
-         <div class="row">
-            <div class="col-lg-12">
-               <h1 class="wow fadeInDown">End Of Coffee<br>
-                  Office inventory tracking app</h1>
-                  <p class="wow fadeInDown">End of coffee helps you track all the office inventory you need! </p>
-                  <div class="bnts wow fadeInUp">
-                     <a href="#" class="btn2">Try Now</a>
-                  </div>
-               </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-lg-12">
+            <h1 class="wow fadeInDown">
+              End Of Coffee<br />
+              Office inventory tracking app
+            </h1>
+            <p class="wow fadeInDown">
+              End of coffee helps you track all the office inventory you need!
+            </p>
+            <div class="bnts wow fadeInUp">
+              <a href="#" class="btn2">Try Now</a>

```

Comment:
only wonder: will it be good if we change href attribute to our localhost app (maybe via some variables...) We don't have a server but locally it will be great to be able to switch between landing page and app

Reply:
I didn't want to hardcoded this - because then we will have to change it everywhere. 
This is just a static file, not going through anything so I can't use .env variables or something like this